### ISSUE TITLE
Suggestion: Publish this as a npm module

### ISSUE THREAD

**Issue by mihaifm** [reactual/hacker-news-favorites-api#7]
It would be useful to be able to use this programmatically (I'm currently copy-pasting your `main.js` code).

**Comment by sbrichardson** [comment]
Good idea @mihaifm! I just published it on [npm as `hnfavs`](https://www.npmjs.com/package/hnfavs).

Few notes:

- I changed the project name in package.json from `hacker-news-favorites-api` to `hnfavs`.
- I moved `x-ray` to peerDependencies, you'll need to add it to your project.
- package.json exports a few different versions browser/main/module etc.
- I only did a quick local test with node after configuring rollup, let me know if you have any issues.

```sh
npm install --save hnfavs x-ray

yarn add hnfavs x-ray
```

```js
// Basic usage
const hnfavs = require('hnfavs')

hnfavs('sbr464').then(console.log)
```
